packages/webapp/src/components/ui/Persona/index.tsx

```diff
@@ -13,9 +13,13 @@ import { useCurrentUser } from '~hooks/api/useCurrentUser'
 import { useNavCallback } from '~hooks/useNavCallback'
 import { ApplicationRoute } from '~types/ApplicationRoute'
 import { useWindowSize } from '~hooks/useWindowSize'
+import { config } from '~utils/config'
```

Comment:
Why are you not using your well made hook?
And the use the nice event listeners you made to set the app online/offline?

Reply:
I figured i'd rework the `useOffline` file so that the `isOffline` state is stored in the store, doing this would allow the state to be pulled without triggering any `addEventListener` calls.  I figured i'd just change the state in the store because the hook would react, but calling the event listener itself also works, i've updated it to follow your advice.